pkg/git/netutils.go

```diff
@@ -39,7 +39,11 @@ func GetAuthFromSecret(url string, creds *corev1.Secret) (transport.AuthMethod,
 		}
 		auth, err := gossh.NewPublicKeys(gitURL.User.Username(), creds.Data[corev1.SSHAuthPrivateKey], "")
 		if err != nil {
-			return nil, err
+			// Try to parse as OpenSSH private key
+			auth, err = gossh.NewPublicKeys(gitURL.User.Username(), creds.Data[corev1.SSHAuthPrivateKey], "")
```

Comment:
Same remark as in `cloner.go`: Why retry the exact same operation as the one that has just failed?